DependencyInjection/Configuration.php

```diff
@@ -132,7 +132,11 @@ public function getConfigTreeBuilder()
         $rootNode = $builder->root('hwi_oauth');
         $rootNode
             ->children()
-                ->scalarNode('firewall_name')->isRequired()->cannotBeEmpty()->end()
```

Comment:
If this option will be removed on next major and produce a BC break, a warning should be triggered to prevent users IMHO.